test/admin/styles.scss

```diff
@@ -1,5 +1,6 @@
 .nav__controls {
     font-family: monospace;
+    background-image: url('/placeholder.png');
```

Comment:
This test case points to a file that seems to exist, would it also work if there is no image?